test/system/projects_test.rb

```diff
@@ -91,8 +91,20 @@ def setup
     fill_in I18n.t(:'activerecord.attributes.namespaces/project_namespace.name'), with: project_name
```

Comment:
Can I get you to add a test for a successful path change? We will want to test if the url updated correctly and assert that the success text was displayed in the flash

Reply:
You are correct! Totally missed it. Thanks